test/compile-test.cc

```diff
@@ -294,7 +294,14 @@ TEST(compile_test, compile_format_string_literal) {
 }
 #endif
 
-#if FMT_CPLUSPLUS >= 202002L || \
+// MSVS 2019 19.29.30145.0 - Support C++20 and OK.
+// MSVS 2022 19.32.31332.0 - compile-test.cc(362,3): fatal error C1001: Internal
```

Comment:
@phprus Have you checked whether a corresponding Visual Studio 2022 issue exists for this problem?

Reply:
Maybe it's https://developercommunity.visualstudio.com/t/internal-compiler-error-c1001-in-compiler-file-da0/1661750 but I'm not sure.